eZ/Bundle/EzPublishCoreBundle/Tests/DependencyInjection/Configuration/Parser/CommonTest.php

```diff
@@ -169,6 +169,25 @@ public function testMiscSettings()
         $this->assertConfigResolverParameterValue('anonymous_user_id', $anonymousUserId, 'ezdemo_site');
     }
 
+    public function testApiKeysSettings()
+    {
+        $key = 'my_key';
+        $this->load(
+            array(
+                'system' => array(
+                    'ezdemo_group' => array(
+                        'api_keys' => array(
+                            'google_maps' => $key,
+                        ),
+                    ),
+                ),
+            )
+        );
+
+        $this->assertConfigResolverParameterValue('api_keys', ['google_maps' => $key], 'ezdemo_site');
```

Comment:
If I remove this line and newly added 
`$contextualizer->setContextualParameter('api_keys', $currentScope, ['google_maps' => $scopeSettings['api_keys']['google_maps']]);`
test passes and I still can see api key in the view rendered on front-end.